site/src/pages/docs/client-factory-builder.mdx

```diff
@@ -0,0 +1,139 @@
+# Customizing a `ClientFactory` with `ClientFactoryBuilder`
+
+A <type://ClientFactory> manages connections and protocol-specific properties.
+You can customize the properties via <type://ClientFactoryBuilder>. For example,
+let's say that you want to increase the connection timeout.
+
+```java
+final ClientFactory factory =
+                ClientFactory.builder()
+                             // Increase the connection timeout to 5 seconds.
+                             .connectTimeoutMillis(5000)
+                             .build());
+final WebClient client = WebClient.builder("http://armeria.com")
+                                  .factory(factory)
+                                  .build();
+client.get("/api").aggregate().join();
+```
+
+The client created with the given <type://ClientFactory> will respect the settings such as connection timeout
+as you defined.
+You might notice that the clients, which are created using the same <type://ClientFactory>,
+share the connection pool. If you don't want to share the connections among the clients,
```

Comment:
```suggestion
share the share connection pool. If you don't want to share the connections among the clients,
```

Reply:
I think we can leave this as it is. Readers can understand the clients spawned by `ClientFactory` share one connection pool.

Perhaps, one connection pool would better rather than the connection pool?

Reply:
sorry this was a typo 😅 meant `same`

Reply:
Otherwise `a` or `one` are also fine, this was just a small grammar point